src/main/interface/msp_box.c

```diff
@@ -95,6 +95,7 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { BOXPARALYZE, "PARALYZE", 45 },
     { BOXGPSRESCUE, "GPS RESCUE", 46 },
     { BOXACROTRAINER, "ACRO TRAINER", 47 },
+    { BOXJUSTSPIN, "JUST SPIN MOTORS", 48 },
```

Comment:
This name is not very intuitive, the 'just' does not do anything to explain to a user what it is. How about 'spin at idle', or 'constant spin motors'?

Reply:
this has been resolved